proto/crescent/liquidity/v1beta1/query.proto

```diff
@@ -261,6 +270,26 @@ message QueryNumMMOrdersResponse {
   uint32 num_market_making_orders = 1;
 }
 
+message QueryTradesRequest {
+  uint64                                pair_id    = 1;
+  string                                orderer    = 2;
+  uint64                                order_id   = 3;
+  cosmos.base.query.v1beta1.PageRequest pagination = 4;
+}
+
+message QueryTradesResponse {
+  repeated Trade                         trades     = 1 [(gogoproto.nullable) = false];
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+message QueryTradeRequest {
+  string id = 1;
+}
+
+message QueryTradeResponse {
+  Trade trade = 1 [(gogoproto.nullable) = false];
+}
+
 //
```

Comment:
I think would be nice if there was a comments for swagger generation or description field that said this endpoint is enabled by the node flag